### ISSUE TITLE
Should I change the loss type of mask branch to F.cross_entropy ,when cfg.MODEL.ROI_BOX_HEAD.NUM_CLASSES equals 2?

### ISSUE THREAD

**Issue by jackshaw** [facebookresearch/maskrcnn-benchmark#302]
## ❓ Questions and Help
When I have just two classes (positive and negative) ,I'm confused wheather the current loss type in mask branch proper for my model? Should I change the loss function to F.cross_entropy?
